jfrog-cli/artifactory/utils/golang/project/dependencies/packageswithdeps.go

```diff
@@ -0,0 +1,256 @@
+package dependencies
+
+import (
+	"fmt"
+	golangutil "github.com/jfrog/jfrog-cli-go/jfrog-cli/artifactory/utils/golang"
+	"github.com/jfrog/jfrog-cli-go/jfrog-cli/utils/config"
+	"github.com/jfrog/jfrog-client-go/httpclient"
+	"github.com/jfrog/jfrog-client-go/utils/errorutils"
+	"github.com/jfrog/jfrog-client-go/utils/io/fileutils"
+	"github.com/jfrog/jfrog-client-go/utils/log"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strings"
+)
+
+// Represent go dependency package when running with deps tidy set to true.
```

Comment:
Suggest rephrasing to:
// **Represents** go dependency when running with deps-tidy set to true.

Reply:
Done